.npmignore

```diff
@@ -2,7 +2,6 @@
 /.coveralls.yml
 /.travis.yml
 /build.xml
-/codesniffer-ruleset.xml
```

Comment:
Please, keep `codesniffer-ruleset.xml` ignored by npm. Codesniffer does not have any official npm package, thus it makes no sense to have this configuration file included. It is only used while having this package as a composer dependency.

Reply:
`codesniffer-ruleset.xml` is :) gone